packages/client/hmi-client/src/services/charts.ts

```diff
@@ -326,7 +326,8 @@ export function createForecastChart(
 	samplingLayer: ForecastChartLayer | null,
 	statisticsLayer: ForecastChartLayer | null,
 	groundTruthLayer: ForecastChartLayer | null,
-	options: ForecastChartOptions
+	options: ForecastChartOptions,
+	summaryLayer?: ForecastChartLayer | null
```

Comment:
Would it be possible to reuse the "statisticsLayer" or the "samplingLayer" rather than creating a new layer definition? Is it a lot of work? 

It just keeps things tighter - because there are a lot of similarities in terms of how people define statistics and summary. e.g. average, max, variation ... are all statistical summarizations of some data 😄 

As an aside, would be better to keep "options" parameter as the last to make the function a little easier to reason with.